sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/github.yaml

```diff
@@ -0,0 +1,78 @@
+provider:
```

Comment:
Nice-to-have: Add links to GitHub documentation on how to rotate and/or revoke tokens.